odinochka.js

```diff
@@ -298,7 +339,7 @@ function initOptions() {
     }
 
 
-    document.getElementsByName("filter")[0].oninput = cssfilter;
+    document.getElementsByName("filter")[0].addEventListener('input', debounce(filterResults, 200))
```

Comment:
feel free to play with this changing `200` ms -> `500` ms if you want to cut down on how often `filterResults` is called even further for performance reasons.